### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `typos --hidden --format brief`

### CODE REVIEW

**Review by Enter-tainer — APPROVED** [review]
nice catch

**Comment by Enter-tainer** [comment]
Hi, please take a look at ci failures

**Comment by kianmeng** [comment]
> Hi, please take a look at ci failures

@Enter-tainer I can't seem to reproduce the error locally?

```
git clone https://github.com/kianmeng/typstyle
cd typstyle
git checkout fix-typos
cargo test
```

```
...
test [typst] tests/assets/unit/code/chained-call-compileable.typ - 0char                            ... ok
test [typst] tests/assets/unit/code/chained-call-compileable.typ - 40char                           ... ok
test [typst] tests/assets/unit/code/chained-call-compileable.typ - 80char                           ... ok
test [typst] tests/assets/unit/code/chained-call-compileable.typ - 120char                          ... ok
test         tests/assets/unit/code/chained-call-compileable.typ - convergence - 40char             ... ok
test         tests/assets/unit/code/chained-call-compileable.typ - convergence - 0char              ... ok
test         tests/assets/unit/code/chained-call-compileable.typ - convergence - 80char             ... ok
test         tests/assets/unit/code/single-arg-closure.typ - output consistency - 40char            ... ok
test         tests/assets/unit/code/show-closure-paren-complex.typ - output consistency - 0char     ... ok
test [typst] tests/assets/unit/code/chained-call-last-indent.typ - 0char                            ... ok
test [typst] tests/assets/unit/code/chained-call-last-indent.typ - 40char                           ... ok
test [typst] tests/assets/unit/code/chained-call-last-indent.typ - 80char                           ... ok
test [typst] tests/assets/unit/code/chained-call-last-indent.typ - 120char                          ... ok
test         tests/assets/unit/code/chained-call-last-indent.typ - convergence - 0char              ... ok
test         tests/assets/unit/code/chained-call-last-indent.typ - convergence - 40char             ... ok
test         tests/assets/unit/code/chained-call-last-indent.typ - convergence - 80char             ... ok
...
test result: ok. 1140 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 46.63s 
```

File: tests/assets/unit/code/chained-call-compilable.typ
Comment (Enter-tainer) [inline tests/assets/unit/code/chained-call-compilable.typ:1]: i guess the ci failure is because of this file rename

Comment (Enter-tainer) [reply tests/assets/unit/code/chained-call-compilable.typ:1]: Since relevant snapshots are not renamed

**Comment by Enter-tainer** [comment]
I'm closing this since it doesn't have update for a while. Feel free to re-open this PR if you want to continue work on it. Thanks!

**Pull request closed without merging**